ui/pages/onboarding-flow/onboarding-flow.test.js

```diff
@@ -29,6 +28,16 @@ import { mockNetworkState } from '../../../test/stub/networks';
 import { FirstTimeFlowType } from '../../../shared/constants/onboarding';
 import OnboardingFlow from './onboarding-flow';
 
+const mockNavigate = jest.fn();
+const mockUseNavigate = jest.fn(() => mockNavigate);
```

Comment:
Whoa what is going on here?  Is this consistent with other uses of `mockNavigate` and `mockUseNavigate`?

Reply:
Good catch !! it was a leftover, removed in 4cd4bc241e86975df1c1f7d3a5ba65ecce5445f5